app/src/main/java/com/example/plantsapp/presentation/ui/profile/ProfileFragment.kt

```diff
@@ -51,8 +56,26 @@ class ProfileFragment : Fragment(R.layout.fragment_profile) {
             )
 
             tvProfileName.text = user.name
+        }
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        inflater.inflate(R.menu.menu_profile_appbar, menu)
+    }
 
-            btnSignOut.isEnabled = true
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        return when (item.itemId) {
+            R.id.action_sign_out -> {
+                viewModel.onSignOutClick()
```

Comment:
also, create alert dialog here like this  (forexample)
![telegram-cloud-photo-size-2-5314527639865768204-y](https://user-images.githubusercontent.com/23121846/145983651-c8170d53-4b77-4679-846c-17c24b34fd40.jpg)

(actually this dialog is little bit wron, because confirmation message should be right button , but anyway )